OlinkAnalyze/vignettes/plate_randomizer.Rmd

```diff
@@ -0,0 +1,116 @@
+library(dplyr)
+library(ggplot2)
+library(kableExtra)
+```
+
+## Introduction
+
+This vignette describes how to use Olink® Analyze to randomize the samples and  plate samples with an option to keep subjects on the same plate. When a study is well randomized the experimental variables can be considered to be evenly distributed across each plate, as well as between the plates of a larger study. What variables to randomize for should be decided for each study as these vary with the study purpose. Variables to randomize for can include e.g. study groups, treatment, sampling time points or demographics. If randomization is not performed true biological variation may be missed or misidentified as e.g. technical variations. In thin vignette, you will learn how to achieve this using the `olink_plate_randomizer()` function. And on top of that, you will also learn how to use `olink_displayPlateLayout()` and `olink_displayPlateDistributions()` to evaluate the performed randomization based on a given grouping variable.
+
+
+```{r Outlier_data, include=FALSE}
+library(OlinkAnalyze)
+data(manifest)
+```
+
+
+## Sample randomization
+
+### Complete randomization
+
+The simplest way to randomize the samples is to perform complete randomization.
+
+```{r complete_randomization, message=FALSE}
+randomized.manifest_a <- olink_plate_randomizer(manifest, seed=12345)
+```
+
+
+### Generate randomization scheme that keeps subjects on the same plate
+
+In the case of multiple samples per subject (e.g. in longitudinal studies), Olink recommends keeping each subject on the same plate to further reduce variation in the data. The individuals should then be distributed so that the remaining experimental variables of interest are evenly distributed across plates. This could be achieved by using the `SubjectColumn` argument. The number of samples on each plate could be specified via the `available.spots` argument. For example, the following code will lead to 78 samples on the first plate, and 30 samples on the second and the third plates. The number of iterations for fitting subjects on the same plate can be set by the `iterations` argument.
+
+
+```{r subjects_randomization, message=FALSE}
+randomized.manifest_b <- olink_plate_randomizer(manifest,SubjectColumn="SubjectID", 
+                                                available.spots=c(78,30,30), 
+                                                iterations = 500,
+                                                seed=12345)
+```
+
+After randomization, the output would be a "tibble" including SampleID, SubjectID etc. assigned to well positions. Columns include same columns as Manifest with additional columns:
+
+  - plate: Plate number
+
+  - column: Column on the plate
+
+  - row: Row on the plate
+
+  - well: Well location on the plate
+
+## Visualization
+
+To illustrate the goodness of randomization, both `olink_displayPlateLayout()` and `olink_displayPlateDistributions` functions could be used.
+### Plate layouts
```

Comment:
need a space between this scentence and ### Plate layouts, otherwise it dont become a title.